app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt

```diff
@@ -27,17 +28,22 @@ class WebViewPresenterImpl @Inject constructor(
 
     private val mainScope: CoroutineScope = CoroutineScope(Dispatchers.Main + Job())
 
+    private var url: URL? = null
+
     override fun onViewReady() {
         mainScope.launch {
-            val url = urlUseCase.getUrl()
+            val oldUrl = url
+            url = urlUseCase.getUrl()
 
-            view.loadUrl(
-                Uri.parse(url.toString())
-                    .buildUpon()
-                    .appendQueryParameter("external_auth", "1")
-                    .build()
-                    .toString()
-            )
+            if (oldUrl != url) {
+                view.loadUrl(
+                    Uri.parse(url.toString())
+                        .buildUpon()
+                        .appendQueryParameter("external_auth", "1")
```

Comment:
Can we add a comment to this line that this is to support versions of Home Assistant before 103 released December 2019. That way we can clean it up in the future :)